src/Compiler/Checking/Expressions/CheckComputationExpressions.fs

```diff
@@ -1988,6 +1988,83 @@ let rec TryTranslateComputationExpression
             Some(translatedCtxt bindExpr)
 
         // 'use! pat = e1 ... in e2' where 'pat' is not a simple name -> error
+        | SynExpr.LetOrUseBang(
```

Comment:
Can you please try to deduplicate the repeated steps with the clause above?
Many parameters are bound on a higher level and not specific to the case (all the cenv and its properties, as well as some constant strings like "Using" and "Bind").

I would want it to be super-obvious to see how this clause and the above differ in their handling.
Right now it is not very apparent.

Reply:
Sure. Will give a try

Reply:
I merged all the 3 clauses where `SynExpr.LetOrUseBang(isUse = true..` in one and also use the new LanguageFeature added.